tensorbay/utility/tbrn.py

```diff
@@ -52,31 +52,32 @@ class TBRNType(Enum):
     """TBRNType defines the type of a TBRN.
 
     It has 7 types:
-    1. `TBRNType.DATASET`::
+
+        1. `TBRNType.DATASET`::
```

Comment:
what format this commit correct?
You need to point it out in the commit message